.github/renovate.json5

```diff
@@ -1,193 +1,158 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "timezone": "Asia/Shanghai",
-  "extends": [
-    ":dependencyDashboard"
-  ],
-  "schedule": [
-    "before 8am on wednesday"
-  ],
-  "enabledManagers": [
-    "github-actions",
-    "cargo",
-    "npm"
-  ],
-  "ignorePaths": [
-    "**/fixtures/**",
-    "**/tests/**",
-    "npm/**",
-    "tests/webpack-examples/**",
-    "tests/webpack-test/**"
-  ],
-  "assignees": [],
-  "packageRules": [
-    {
-			"matchPackagePatterns": ["*"],
-			"semanticCommitType": "chore",
+	$schema: "https://docs.renovatebot.com/renovate-schema.json",
+	timezone: "Asia/Shanghai",
+	extends: [":dependencyDashboard"],
+	schedule: ["before 8am on wednesday"],
+	enabledManagers: ["github-actions", "cargo", "npm"],
```

Comment:
We have so many pnpm bump PRs, can we reduce then? https://github.com/web-infra-dev/rspack/pull/8256

For example, only bump pnpm when there is a new minor release.

Reply:
done by add pnpm group